culture.md

```diff
@@ -0,0 +1,31 @@
+Our Culture
+===========
+
+the lgbtq in technology slack is a big place. We do not all agree on anything, but there's a rough consensus around some things. The [code of conduct](coc.md) is uncontroversial at this point, so it's worth a read.
+
+We have active moderators, generally willing to intervene. If we see racist comments, even subtle, we intend to call them out: this is not an attack on the speaker, but a correction. These comments affect members of the slack, and this affirms that their presence is welcome. This is the same for transphobic or ciscentric comments, for similar reasons. This goes beyond explicitly offensive statements, and we encourage nuance and precision -- and being aware that the membership of the slack is in fact quite diverse, and we are speaking to people both like _and unlike_ ourselves.
+
+Also in an effort to keep people from being spoken over, there is a statistics bot that tracks the rough demographics of groups and reminds us that the proportion of people speaking.
+
+There are a huge number of channels, over 250 at time of this writing. A great many are identity-exclusive: #trans is space exclusively for trans folks; #not-men is space exclusively for people who are not men; in general we seek to support and make space for groups that get talked over and ignored. There are channels for geographic areas, some large and some small.
+
+We support autonomy: most channels were simply created by someone who decided it was a good idea. We frown on doing things to people without their consent.
+
+Some channels have a `-unfiltered` counterpart; it's completely acceptable to suggest moving a conversation to the `-unfiltered` channel, and completely normal to leave that channel if upsetting conversation is there.  this leaves identity channels free to keep their membership so people can find each other, without having to be in conversations they are not equipped for.
+
+It is common to join many channels, and conversations can span multiple channels. Our community is pretty cohesive, and we find the same people in many channels. This is different in IRC, where each channel forms its own community, ours are much more expansive. This changs how conversation flows, and is worth being aware of.
+
+We encourage sexually explicit pictures to be posted as URLs, not uploaded as files, as the 'all files' tab is visible by people regardless of their channel membership. Upsetting images should go further and some effort made to hide them from Slack's automatic link expansion.
+
+Selfies are encouraged. We get to know each other, we can show love for ourselves and it's not considered narcissism.
```

Comment:
"The #selfies channel is for selfies, and these are encouraged as we get to know each other, we can show love for ourselves and it's not narcissism."